sale_automatic_workflow/tests/test_automatic_workflow.py

```diff
@@ -7,12 +7,10 @@
 from freezegun import freeze_time
 
 from odoo import fields
-from odoo.tests import tagged
 
 from .common import TestAutomaticWorkflowMixin, TestCommon
```

Comment:
I would adapt [`TestCommon`](https://github.com/OCA/sale-workflow/blob/18.0/sale_automatic_workflow/tests/common.py#L11-L15) so it uses [`BaseCommon`](https://github.com/odoo/odoo/blob/18.0/odoo/addons/base/tests/common.py#L20) instead of `TransactionCase`

> Use BaseCommon as base test class to reduce overhead from tracking and boost the test suite.